client/src/components/App/News.js

```diff
@@ -0,0 +1,126 @@
+import React, { PureComponent, Fragment } from 'react';
+import { Button, Divider, List, ListItem, Paper, Popover, Tooltip, Typography, withStyles } from '@material-ui/core';
+import { RssFeed } from '@material-ui/icons';
+import { NEWS_ENDPOINT } from '../../api/endpoints';
+import { Skeleton } from '@material-ui/lab';
+import moment from 'moment-timezone';
+
+const styles = (theme) => ({
+    list: {
+        width: theme.spacing(40),
+        maxHeight: theme.spacing(40),
+        overflow: 'auto',
+    },
+    listItem: {
+        display: 'flex',
+        flexDirection: 'column',
+    },
+    skeleton: {
+        padding: '4px',
+    },
+});
+
+class News extends PureComponent {
+    state = {
+        anchorEl: null,
+        newsItems: null,
+        loading: true,
+    };
+
+    componentDidMount = async () => {
+        try {
+            const data = await fetch(NEWS_ENDPOINT);
+            const json = await data.json();
+            this.setState({ newsItems: json.news, loading: false });
+        } catch (e) {
+            console.error('Error loading news items:', e);
+            this.setState({ newsItems: null, loading: false });
+        }
+    };
+
+    getNewsItems = () => {
+        const { classes } = this.props;
+        if (this.state.loading === false && this.state.newsItems !== null) {
```

Comment:
```suggestion
        if (this.state.loading === false && this.state.newsItems !== null && this.state.newsItems.length) {
```
Suggestion to fail the check if there are no items in newItems. We don't want to return an empty module.